src/routes/evento.routes.js

```diff
@@ -0,0 +1,32 @@
+const { Router } = require('express')
+const Evento = require('../models/evento')
+const verificarToken = require('../tools/verificarToken')
+
+
+const router = Router()
+
+router.get('/eventos', async(req, resp) => {
+    const eventos = await Evento.find()
+    return resp.json({
+        msg: '200 OK',
+        eventos
+    })
+})
+router.post('/evento', async(req, res) => {
+    const { _id } = req.body
+    const evento = await Evento.findById({ _id })
+    return res.json({
+        msg: '200 OK',
+        evento
+    })
+})
+router.get('/mis-eventos', verificarToken, async(req, res) => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Backend/pullRequest?prid=7138706)
```suggestion
router.get("/mis-eventos", verificarToken, async(req, res) => {
```